test_sslify.py

```diff
@@ -61,3 +61,10 @@ def test_hsts_subdomains():
     assert status == '200 OK'
     assert headers['Strict-Transport-Security'] == 'max-age=31536000; includeSubDomains'
 
+def test_hsts_preload():
+    app = sslify(testapp.test_app, preload=True)
+    env = create_environ()
+    env['wsgi.url_scheme'] = 'https'
+    app_iter, status, headers = run_wsgi_app(app, env)
+    assert status == '200 OK'
+    assert 'preload' in headers['Strict-Transport-Security']
```

Comment:
I think it might be best to validate the entire header, rather than just that the string exists in it? (Like the test above does)

For example the header `max-age=123 preload` would pass this test, but be ignored by browsers according to the spec:
https://tools.ietf.org/html/rfc6797#section-6.1

In addition it may be worth also adding a second test, that confirms both `subdomains=True` and `preload=True` can be set at the same time (eg `test_hsts_all_options_combined()`).